tests/migration/postcopy.go

```diff
@@ -257,22 +244,25 @@ var _ = SIGMigrationDescribe("VM Post Copy Live Migration", func() {
 						}, 120*time.Second, 1*time.Second).Should(Succeed(), "Virt handler should come online")
 					}
 					It("should make sure that VM restarts after failure", func() {
-
 						By("creating a large VM with RunStrategyRerunOnFailure")
-						vm := createLargeVirtualMachine()
+						vmi := libvmifact.NewFedora(
+							append(libnet.WithMasqueradeNetworking(),
```

Comment:
requires rebase now that `libnet.WithMasqueradeNetworking()` returns one option.

Reply:
Done :)